pkg/runner/simple.go

```diff
@@ -93,11 +93,17 @@ func (r *SimpleRunner) Execute(req *ffuf.Request) (ffuf.Response, error) {
 	if err != nil {
 		return ffuf.Response{}, err
 	}
+
 	// Add user agent string if not defined
 	if _, ok := req.Headers["User-Agent"]; !ok {
-		req.Headers["User-Agent"] = fmt.Sprintf("%s v%s", "Fuzz Faster U Fool", ffuf.VERSION)
+		if _, ok := req.Headers["User-agent"]; !ok {
```

Comment:
HTTP header names are case insensitive as defined in [RFC 2616 section 4.2](https://ietf.org/rfc/rfc2616.html#section-4.2)
Because of this we should account for all possible names, including `uSEr-agEnt` etc. 

The proper way to handle it would be iterating through `req.Headers` keys, and check if any of them contain `user-agent` by using case-insensitive matching provided by [strings.EqualFold()](https://golang.org/pkg/strings/#EqualFold)